core/arch/arm/kernel/ssvce_a32.S

```diff
@@ -49,80 +49,45 @@
  */
 
 
-/*
- * void secure_mmu_unifiedtlbinvall(void);
- */
-FUNC secure_mmu_unifiedtlbinvall , :
+/* void tlbi_all(void); */
+FUNC tlbi_all , :
 UNWIND(	.fnstart)
 
-	dsb	/* Ensure visibility of the update to translation table walks */
-	write_tlbiallis
+	dsb	ishst		/* Sync with table update */
+	write_tlbiallis 	/* Invalidate TLBs */
+	dsb	ish		/* Sync with tlb invalidation completion */
+	isb			/* Sync execution on tlb update */
+	bx	lr
 
-	DSB
-	ISB
-
-	MOV     PC, LR
 UNWIND(	.fnend)
-END_FUNC secure_mmu_unifiedtlbinvall
+END_FUNC tlbi_all
 
-/*
- * void secure_mmu_unifiedtlbinvbymva(mva);
- *
- * Combine VA and current ASID, and invalidate matching TLB
- */
-FUNC secure_mmu_unifiedtlbinvbymva , :
+/* void tlbi_mva_allasid(vaddr_t mva); */
+FUNC tlbi_mva_allasid , :
 UNWIND(	.fnstart)
 
```

Comment:
Empty line not needed, please fix the other functions you're touching in this file too.